EventsExpress/ClientApp/src/components/event/event-item-view.js

```diff
@@ -139,14 +140,30 @@ export default class EventItemView extends Component {
         if (visitor !== undefined) {
             switch (visitor.userStatusEvent) {
                 case 0:
```

Comment:
Please create an enum for these values like here: https://www.sohamkamani.com/blog/2017/08/21/enums-in-javascript/